src/apps/bristleback_apps/bristleback_apps_common/app_main.cpp

```diff
@@ -322,6 +322,10 @@ static void defaultTask(void *parameters) {
   // Inhibit low power mode during boot process
   lpmPeripheralActive(LPM_BOOT);
 
+  // If we ever include the SWWDG, we need to have the memfault
+  // platform boot before starting the watchdog task
+  memfault_platform_boot();
```

Comment:
as the comment states, I used the software watchdog for debugging this build and if it is ever enabled again in the future, this needs to be called before the watchdog task is started.